modules/swagger-core/src/test/java/io/swagger/jackson/XMLInfoTest.java

```diff
@@ -34,14 +35,19 @@ public void testSimple() throws Exception {
         assertNotNull(xml);
         assertEquals(xml.getName(), "xmlDecoratedBean");
 
-        final Property property = impl.getProperties().get("items");
+        // Cast it to an array property
+        final ArrayProperty property = (ArrayProperty)impl.getProperties().get("items");
         assertNotNull(property);
         final Xml propertyXml = property.getXml();
-
         assertNotNull(propertyXml);
-        assertEquals(propertyXml.getName(), "item");
+        // Check the name of array property
+        assertEquals(propertyXml.getName(), "items");
         assertTrue(propertyXml.getWrapped());
-
+        // Get the xml for items for the array property
+        final Xml itemsXml = property.getItems().getXml();
```

Comment:
In `XmlDecoratedBean` the `items` should be represented eventually as:

```yaml
items:
  type: array
  items:
    type: string
    xml:
      name: item
  xml:
    name: items
    wrapped: true
```

Two notes:
1. the `name: items` is not required in this case, because it's already the name of the property in any case. We could add a check if the value of the `@XmlElementWrapper` is the same as the field name, then not add the `name` to the `xml` object.
2. The example itself is confusing to a reader because it uses the name `items` for an array, which in JSON Schema uses the property `items` - just making it difficult to follow.